### PR TITLE
Run test262 on windows

### PR DESCRIPTION
**Pull request by ksh8281**

### CODE REVIEW

File: src/runtime/Object.cpp [hunk 1]
```diff
@@ -999,6 +999,8 @@ struct CompareIndexItem {
 template <typename ResultType, typename ResultBinder>
 static ResultType objectOwnPropertyKeys(ExecutionState& state, Object* self)
 {
+    // TODO turn-on gc if replace std::multiset to another gc-well-supported type
+    GC_disable();
```
Comment (clover2123) [inline src/runtime/Object.cpp:1003]: Why is `GC` disabled here?

**Comment by clover2123** [comment]
BTW some test files of `robin_map` are included such as `third_party/robin_map/tests/main.cpp`.
Are these files also required?

File: src/runtime/Object.cpp (hunk 1 above)
Comment (ksh8281) [reply src/runtime/Object.cpp:1003]: Because bdwgc may not cover std::multiset also

**Comment by ksh8281** [comment]
@clover2123 I would delete test files of robin_map

File: src/runtime/Object.cpp (hunk 1 above)
Comment (clover2123) [reply src/runtime/Object.cpp]: I got it that `std::multiset` could be another issue

**Review by clover2123 — APPROVED** [review]
Great patch 👍

**Pull request merged**